doc/Pacemaker_Explained/en-US/Ch-Advanced-Options.txt

```diff
@@ -627,6 +627,35 @@ The +ipmiservicelogd+ daemon sets node health attributes based on IPMI
 values (the +ocf:pacemaker:SystemHealth+ resource agent can be used to manage
 the daemon as a cluster resource).
 
+In order to take advantage of this feature - firstly add the resource to your cluster, preferably as a cloned resource to constatnly measure health on all nodes:
+
+=====
+[source,XML]
+------------
+<clone id="resHealthCPU-clone">
+  <primitive class="ocf" id="resHealthCPU" provider="pacemaker" type="CPUIOWaitHealth">
```

Comment:
The type attribute here, the name of the resource agent file, and the &lt;resource-agent&gt; name attribute should all be identical. Looking at the current names, I'm thinking "HealthIOWait" would be simple and descriptive.

Reply:
Changed, shall I also rename the bash script to match it ?

Reply:
Renamed it anyways.